Fix comparison of translation parts in MSG type identification (#382)
spglib/spglib: test/README.md

            ...
            "includePath": [
                "${workspaceFolder}/**",
                "${workspaceFolder}/cmake-build-release-gcc/_deps/googletest-src/googletest/include",
                "${workspaceFolder}/cmake-build-release-llvm/_deps/googletest-src/googletest/include"
            ],
            ...
        }
    ]
}
```

> The below `launch.json` configures debugger in running a single test:

Reviewer: ```suggestion
The `launch.json` below runs a single test with a debugger:
```

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug Launch(lldb): #381",
            "type": "lldb",
            "request": "launch",

>             "program": "${workspaceFolder}/build/test/test_suite",

Reviewer: Should also work with calling `ctest`. That is more helpful because it also handles `pytest`.

Author: Which `ctest` to specify? Does it locate under `${workspaceFolder}`?

Reviewer: I mean you can do `"program": "ctest"` with `--test-dir ./build` and then use `-R`/`-L` etc. to select the tests instead of running the `test_suite`.

Author: I made a follow-up issue: https://github.com/spglib/spglib/issues/383
